cluster-components/nginx-ingress/helm-values.yml

```diff
@@ -0,0 +1,39 @@
+controller:
+  replicaCount: 3
+
+  # Add a snippet to each nginx server{} block, forcing HTTP->HTTPS
+  # redirection regardless of Ingress rule config
+  config:
+    server-snippet: |
+      if ($http_x_forwarded_proto != 'https') {
+        return 308 https://$host$request_uri;
+      }
+
+  # nginx-ingress's ServiceType=LoadBalancer by default, so we're mainly
+  # configuring an ELB here
+  service:
+    annotations:
+      # Have `external-dns` create a Route53 record pointing to this ELB
+      external-dns.alpha.kubernetes.io/hostname: "*.apps.cloud-platforms-sandbox.k8s.integration.dsd.io"
+
+      # Attach pre-existing ACM cert
+      service.beta.kubernetes.io/aws-load-balancer-ssl-cert: "arn:aws:acm:eu-west-1:926803513772:certificate/d737162e-7dc3-434b-ba0c-b0d9fcb53e7c"
```

Comment:
Do we care if this specific cert goes away? Is this the cert that's created in c949106? If so, can this get the ARN automatically somehow (assuming that's not super overkill…)?

Reply:
Yes we care - without the SSL cert we can't do SSL.

Yes it's the cert created in this PR.

No we can't create it automatically - neither Kubernetes nor nginx-ingress have any mechanism for ACM provisioning or attachment other than passing the full ARN of a pre-existing ELB.